test/mocha.js

```diff
@@ -0,0 +1,11 @@
+process.env.NODE_ENV = 'test';
```

Comment:
Move this to `mocha.opts` as `--env` flag cause that is where people would typically look for it. Add `use strict` pragma on the top of the file.

Reply:
Could you give me the syntax of a `--env` flag with mocha? I'm looking for it's documentation and have thrown some stuff at the wall, but nothing seems to work.

Reply:
I don't think there _is_ an `--env` flag: https://github.com/mochajs/mocha/issues/185. I am good with this or setting it via a variable in the npm invocation string.